Add Microsoft SNDS metrics (#6210)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/analytics/deliverability_center.md

## Integrating with Microsoft Smart Network Data Servies (SNDS)

{% alert important %}
Integrating with Microsoft SNDS in the Deliverability Center is currently in early access. Contact your account manager if you're interested in participating in this early access.
{% endalert %}

If Microsoft is your main mailbox provider, you can use this integration to access and view your Microsoft reputation data. This way, you can monitor the health of your IPs to help determine how your emails are being received.

1. Go to **Analytics** > **Email Performance**.
2. Select the **Deliverability Center** tab.
3. Click **Connect with Microsoft SNDS**. 
4. Select your Microsoft account, and click **Allow** to allow Braze to view email traffic metrics for the domains registered.

Your verified domains will display in the Deliverability Center.

![][8]

### Metrics and definitions

The following metrics apply to Microsoft SNDS.

#### Recipients

This metric refers to the number of recipients on messages transmitted by the IP.

#### DATA commands

> This metric tracks the number of DATA commands sent by the IP. DATA commands DATA commands are part of the SMTP protocol used to send mail.

Reviewer: ```suggestion
This metric tracks the number of DATA commands sent by the IP. DATA commands are part of the SMTP protocol used to send mail.
```

#### Filter results

Refer to this table to understand the filter results 

| Result | Definition |
| ----- | ---------- |
| Green | Judged to be spam by Microsoft’s spam filter up to 10% of the given time frame. |
| Yellow | Judged to be spam by Microsoft’s spam filter between 10% and 90% of the given time frame. |

> | Red | Judged to be spam by Microsoft’s spam filter by more than 90% of the given time frame.| 

Reviewer: ```suggestion
| Red | Judged to be spam by Microsoft’s spam filter up to more than 90% of the given time frame.| 
```
OR keep as is and tweak the row for Green:

| Green | Judged to be spam by Microsoft’s spam filter by up to 10% of the given time frame. |

{: .reset-td-br-1 .reset-td-br-2}

#### Complaint rate

> This is the fraction of the time that a message received from the IP is complained about by a Hotmail or Windows Live user during the activity period.  Users have the option of reporting almost all messages as junk via the web user interface. 

Reviewer: ```suggestion
This is the fraction of the time that a message received from the IP is complained about by a Hotmail or Windows Live user during the activity period. Users have the option of reporting almost all messages as junk via the web user interface. 
```

To calculate the complaint rate, divide the number of complaints by the number of message recipients.  

| Result | Definition |
| ----- | ---------- |
| Less than 0.3% | The ideal complaint rate. |
| More than 0.3% | Review your sign-up process, and ensure your unsubscribe link is working. Also, consider whether mail could be better personalized to your audience. |
| More than 100% | Note that SNDS displays complaints for the day they were reported, not retroactively against the day the complained-about mail was delivered. | 
{: .reset-td-br-1 .reset-td-br-2}

#### Spam trap hits

> Spam trap hits are the number of messages sent to "trap accounts," which are accounts maintained by Outlook.com that don't solicit any mail. It's likely that any messages sent to these trap accounts are considered spam, so it's important to monitor this metric to ensure that it's low, which means the messages aren't sent to these accounts and being sent to actual accounts instead.

Reviewer: ```suggestion
Spam trap hits are the number of messages sent to "trap accounts," which are accounts maintained by Outlook.com that don't solicit any mail. It's likely that any messages sent to these trap accounts are considered spam, so it's important to monitor this metric to make sure that it's low, which means the messages aren't sent to these accounts and are being sent to actual accounts instead.
```
